simulation/simulation.py

```diff
@@ -2,15 +2,15 @@
 
 class Simulation:
   def __init__(self, bank: str, financing_value: float, installments_number: int, age: int):
-    self.bank = bank.upper()
+    self.bank = bank
     self.financing_value = financing_value
     self.installments_number = installments_number
     self.age = age
 
   def run(self):
       try:
-        bank = Banks[self.bank].value
-      except ValueError:
+        bank = Banks[self.bank.upper()].value
+      except KeyError:
```

Comment:
Why did you use `KeyError`?

Reply:
if the key does not match any member name in enum, python throws KeyError.

Reply:
Fair enough.